statsmodels/stats/contingency_tables.py

```diff
@@ -0,0 +1,1312 @@
+
+        Arguments
+        ---------
+        null : float
+            The null value of the log odds ratio.
+        """
+
+        zscore = (self.log_oddsratio - null) / self.log_oddsratio_se
+        pvalue = 2 * stats.norm.cdf(-np.abs(zscore))
+        return pvalue
+
+
+    def log_oddsratio_confint(self, alpha=0.05):
+        """
+        A confidence level for the log odds ratio.
+
+        Parameters
+        ----------
+        alpha : float
+            `1 - alpha` is the nominal coverage probability of the
+            confidence interval.
+        """
+
+        f = -stats.norm.ppf(alpha / 2)
+        lor = self.log_oddsratio
+        se = self.log_oddsratio_se
+        lcb = lor - f * se
+        ucb = lor + f * se
+        return lcb, ucb
+
+
+    def oddsratio_confint(self, alpha=0.05):
+        """
+        A confidence interval for the odds ratio.
+
+        Parameters
+        ----------
+        alpha : float
+            `1 - alpha` is the nominal coverage probability of the
+            confidence interval.
+        """
+        lcb, ucb = self.log_oddsratio_confint(alpha)
+        return np.exp(lcb), np.exp(ucb)
+
+
+    @cache_readonly
+    def riskratio(self):
+        # docstring for cached attributes in init above
+
+        p = self.table[:, 0] / self.table.sum(1)
+        return p[0] / p[1]
+
+
+    @cache_readonly
+    def log_riskratio(self):
+        # docstring for cached attributes in init above
+
+        return np.log(self.riskratio)
+
+
+    @cache_readonly
+    def log_riskratio_se(self):
+        # docstring for cached attributes in init above
+
+        n = self.table.sum(1)
+        p = self.table[:, 0] / n
+        va = np.sum((1 - p) / (n*p))
+        return np.sqrt(va)
+
+
+    def riskratio_pvalue(self, null=1):
+        """
+        p-value for a hypothesis test about the risk ratio.
+
+        Arguments
+        ---------
+        null : float
+            The null value of the risk ratio.
+        """
+
+        return self.log_riskratio_pvalue(np.log(null))
+
+
+    def log_riskratio_pvalue(self, null=0):
+        """
+        p-value for a hypothesis test about the log risk ratio.
+
+        Arguments
+        ---------
+        null : float
+            The null value of the log risk ratio.
+        """
+
+        zscore = (self.log_riskratio - null) / self.log_riskratio_se
+        pvalue = 2 * stats.norm.cdf(-np.abs(zscore))
+        return pvalue
+
+
+    def log_riskratio_confint(self, alpha=0.05):
+        """
+        A confidence interval for the log risk ratio.
+
+        Parameters
+        ----------
+        alpha : float
+            `1 - alpha` is the nominal coverage probability of the
+            confidence interval.
+        """
+        f = -stats.norm.ppf(alpha / 2)
+        lrr = self.log_riskratio
+        se = self.log_riskratio_se
+        lcb = lrr - f * se
+        ucb = lrr + f * se
+        return lcb, ucb
+
+
+    def riskratio_confint(self, alpha=0.05):
```

Comment:
Sure, any suggestions on the name?  One of these, maybe?

method="asymptotic"
method="clt"